ideascube/serveradmin/catalog.py

```diff
@@ -604,9 +606,19 @@ def list_nothandled(self, ids):
 
         return sorted(pkgs, key=attrgetter('id'))
 
+    def _update_displayed_packages_on_home(self, *, to_remove_ids=None, to_add_ids=None):
```

Comment:
> In def foo(bar=None): ... bar is not a keyword argument, it's a argument with a default value.

The difference is academic. In practice, there just isn't one.

> With def _update_displayed_packages_on_home(self, to_remove_ids=None, to_add_ids=None) you can write :
> 
```
self._update_displayed_packages_on_home(a_list)
self._update_displayed_packages_on_home(a_list, another_list)
```

And what I'm saying is that this is a terrible idea and I wouldn't accept code like that during a review.

Just because it is possible to do something doesn't mean it's a good idea.

> If you don't what a user to use foo('value') cause you think he should use a kwarg, explicit it !

And I've always considered that defining the signature of a function with kwargs means you should call it as kwargs. The order might change, new ones might be added, calling it as kwargs is the only safe and unconfusing way to call it.

The fact that Python accepts calling with positional arguments a function defined with kwargs is a deisgn mistake in the language IMHO. The behaviour with the `*` is what should be the default.

If you strongly feel we should use `*` to make it explicit, then I suggest we start using `*` everywhere and outright ban the alternative. Not to be done in this PR, but it's a discussion we should have for the future. :wink: